File: CS50x 2021/Week 1/5- PSet 1 Cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    int c = 0;

    //Get owe...
    printf("Write down the amount you owe to calculate the number of coins like ($.¢)...\n");
    float o;
    do
    {
        o = get_float("Owed: ");

    }
    while (0 > o);
    //rounding number.
    int ce = round(o * 100);
    //Quarter.
    while (25 <= ce)
    {
        ce = ce - 25;
        c++;
    }
    //Dime.
    while (10 <= ce)
    {
        ce = ce - 10;
        c++;
    }
    //Nickel.
    while (5 <= ce)
    {
        ce = ce - 5;
        c++;
    }
    //Penny.
    while (1 <= ce)
    {
        ce = ce - 1;
        c++;
    }
    //End!
    printf("%i\n", c);
}

File: CS50x 2021/Week 6/5- PSet 6 Sentimental ' Cash/cash.py
from cs50 import get_float

c = 0
# Get owe...
print("Write down the amount you owe to calculate the number of coins like ($.¢)... ")
o = 0
while True:
    o = get_float("Owed: ")
    if o > 0:
        break
# rounding number.
ce = round(o * 100)
# Quarter.
while 25 <= ce:
    ce -= 25
    c += 1
# Dime.
while 10 <= ce:
    ce -= 10
    c += 1
# Nickel.
while 5 <= ce:
    ce -= 5
    c += 1
# Penny.
while 1 <= ce:
    ce -= 1
    c += 1
# End!
print(c)